tests/Settings.ts

```diff
@@ -8,6 +8,9 @@ export class Settings {
     public static SpeechRegion: string = "<<YOUR_REGION>>";
     public static SpeechEndpoint: string;
 
+    public static VoiceSignatureEnrollmentEndpoint: string = "wss://10.91.185.105:8446/speech/voicesignature?&language=en-us";
```

Comment:
>10.91.185.105:8446 [](start = 68, length = 18)

Can you open a task so we replace this with a public URL once the endpoint is in prod?

Reply:
Bug 2214615: JavaScript: Add VoiceSignatureEnrollment PPE endpoint into tests/Settings.ts